TODOS.md

```diff
@@ -1,11 +1,12 @@
-| packages/product-json-to-csv/test/writer.spec.js | 263 | the "unzip" package fires finish event before entry events
-| packages/product-json-to-csv/test/writer.spec.js | 308 | the "unzip" package fires finish event before entry events
-| packages/sync-actions/src/utils/diffpatcher.js | 3 | create an issue here https://github.com/benjamine/jsondiffpatch/issues/new
-| /Users/deriksson/Sites/commercetools/nodejs/packages/custom-objects-importer/src/main.js | 127 | remove `FlowFixMe` when [this](https://github.com/facebook/flow/issues/5294) issue is fixed
-| /Users/deriksson/Sites/commercetools/nodejs/packages/product-json-to-csv/test/writer.spec.js | 263 | the "unzip" package fires finish event before entry events
-| /Users/deriksson/Sites/commercetools/nodejs/packages/product-json-to-csv/test/writer.spec.js | 308 | the "unzip" package fires finish event before entry events
+
+| Filename                                                                                     | line # | TODO                                                                                        |
+| :------------------------------------------------------------------------------------------- | :----: | :------------------------------------------------------------------------------------------ |
+| packages/api-request-builder/src/create-service.js                                           |  134   | this can lead to invalid URIs as getIdOrKey can return                                      |
+| packages/custom-objects-importer/src/main.js                                                 |  127   | remove `FlowFixMe` when [this](https://github.com/facebook/flow/issues/5294) issue is fixed |
+| packages/product-json-to-csv/test/writer.spec.js                                             |  263   | the "unzip" package fires finish event before entry events                                  |
+| packages/product-json-to-csv/test/writer.spec.js                                             |  308   | the "unzip" package fires finish event before entry events                                  |
+| packages/sync-actions/src/utils/diffpatcher.js                                               |   3    | create an issue here https://github.com/benjamine/jsondiffpatch/issues/new                  |
+| /Users/deriksson/Sites/commercetools/nodejs/packages/custom-objects-importer/src/main.js     |  127   | remove `FlowFixMe` when [this](https://github.com/facebook/flow/issues/5294) issue is fixed |
+| /Users/deriksson/Sites/commercetools/nodejs/packages/product-json-to-csv/test/writer.spec.js |  263   | the "unzip" package fires finish event before entry events                                  |
+| /Users/deriksson/Sites/commercetools/nodejs/packages/product-json-to-csv/test/writer.spec.js |  308   | the "unzip" package fires finish event before entry events                                  |
```

Comment:
should we keep `Users/<name>` info in this list? 😆

Reply:
hehe... nvm this one. Looks like my integration of `leasot` some time ago doesn't work properly 😇

Reply:
Good catch and I'm fine with fixing @daern91 issues 😄